MODULE.bazel.lock

```diff
@@ -0,0 +1,1273 @@
+{
```

Comment:
This should be gitignored for now. Maybe in bazel `7.2` we can check-it in. This is because there will be many git conflicts if we leave this file here.

Reply:
I would like to remove these, but `buildifier` seems to check if these files exist (see [last CI run](https://github.com/woven-planet/go-zserio/actions/runs/7913527328/job/21601371227)).
Is there a way to exclude this from the `buildifier` step?

Reply:
Then we should add `common --noenable_bzlmod` to `.bazelrc`. This should avoid the automatic creation of `MODULE.bazel*` files.

Reply:
Done!